src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -855,6 +855,60 @@ await using (var context = InMemoryApplicationDbContext(contextId))
             }
         }
 
+        [Fact]
+        public async Task CreatePublication_NoContactTelNo()
+        {
+            var topic = new Topic
+            {
+                Title = "Test topic",
+                Theme = new Theme
+                {
+                    Title = "Test theme",
+                },
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                context.Add(topic);
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var publicationService = BuildPublicationService(context);
+
+                // Service method under test
+                var result = await publicationService.CreatePublication(
+                    new PublicationCreateRequest
+                    {
+                        Title = "Test publication",
+                        Summary = "Test summary",
+                        Contact = new ContactSaveViewModel
+                        {
+                            ContactName = "John Smith",
+                            ContactTelNo = "",
+                            TeamName = "Test team",
+                            TeamEmail = "john.smith@test.com",
+                        },
+                        TopicId = topic.Id
+                    }
+                );
+
+                var publicationViewModel = result.AssertRight();
+                Assert.Equal("Test publication", publicationViewModel.Title);
+
+                Assert.Equal("John Smith", publicationViewModel.Contact.ContactName);
+                Assert.Null(publicationViewModel.Contact.ContactTelNo);
+
+                var createdPublication = await context.Publications.FindAsync(publicationViewModel.Id);
+                Assert.NotNull(createdPublication);
+                Assert.Equal("John Smith", createdPublication.Contact.ContactName);
+                Assert.Null(createdPublication.Contact.ContactTelNo);
```

Comment:
I think your missing an `Include` on `Contact` here. I would also use `First` not `FindAsync` because then the test will fail due to the exception if it's not found, avoiding the extra `NotNull` assertion, and I'm wary of using `FindAsync` because it uses cached results.

Reply:
Interesting - I wonder if it was because `FindAsync` is cached that it passed despite the missing `Include`.

I've made the same change in the test I copied this from.